docs/usage.md

```diff
@@ -0,0 +1,120 @@
+# About
+
+The Data Commons import tool is used to analyze and debug files that are developed
+in the process of importing new datasets to the Data Commons Knowledge Graph.
+
+It can;
+- debug schema (.mcf) files
+- generate instance MCF from template MCF (TMCF) and corresponding CSV tables
+- resolve references against the Data Commons API
+- generate statistics validation reports to spot check time series data
+
+The tool is actively used for all data imports that are included in the Data Commons Knowledge Graph.
+It is under active development, including feature additions and bug fixes.
+
+The tool is a command line application built with Java. See below for usage instructions.
+
+# Usage
+
+Use the import tool from the command line, like so:
+
+  ```bash
+  java -jar <path-to-jar> <mode> <list of mcf/tmcf/csv files>
+  ```
+
+Hint: it can be useful to create an alias for the jar file, such as:
+  ```bash
+  alias dc-import='java -jar <path-to-jar>'
+  ```
+
+This is the form that will be used in the rest of the documentation.
+
+Hint: to access a concise explanation of usage modes and flags, run
+`dc-import --help`
+
+## Usage Modes
+
+In `lint` mode, the import tool validates the artifacts produced for addition to Data Commons. These artifacts include [instance MCF files](https://github.com/datacommonsorg/data/blob/master/docs/mcf_format.md#instance-mcf) and pairs of [template MCF (TMCF)](https://github.com/datacommonsorg/data/blob/master/docs/mcf_format.md#instance-mcf) and corresponding CSV files.
+
+In `genmcf` mode, the import tool produces instance MCF files from a pair of TMCF file, and its associated CSV files. This mode performs all validations that the  `lint` mode would have performed.
+
+All flags below apply to both the modes.
+
+### Lint Mode (`lint`)
+To run the tool in lint mode, use:
+  ```bash
+  dc-import lint <list of mcf files>
+  ```
+
+For example;
+  ```bash
+  dc-import lint auto_generated_stat_vars.mcf manual_schema.mcf
+  ```
+
+### Instance MCF Generation Mode (`genmcf`)
+To run the tool in genmcf mode, use:
+  ```bash
+  dc-import genmcf <list of csv/tmcf files>
+  ```
+
+Note that in this mode, schema file(s) can be optionally passed. This is required to resolve references to newly introduced schema nodes.
+
+  For example;
+  ```bash
+  dc-import lint my_dataset_mapping.tmcf my_dataset_table.csv  auto_generated_stat_vars.mcf manual_schema.mcf
```

Comment:
Can we pick some existing example files from the current import?

Reply:
using test files embedded in the repo